### ISSUE TITLE
Refactor global config

### ISSUE THREAD

**Issue by ChihChengLiang** [thehubbleproject/hubble-commander#64]
## What's wrong

We now have our node configuration as a global variable located at

https://github.com/thehubbleproject/hubble-commander/blob/9eaaf4ce6f1a6362f61c321789afac339bb366d0/config/config.go#L31

And when we load a new config we override the variable here

https://github.com/thehubbleproject/hubble-commander/blob/9eaaf4ce6f1a6362f61c321789afac339bb366d0/config/config.go#L100-L107

This will make testing harder and make the codebase error-prone

## How can we fix it

We should let functions take the config as arguments. For example, when we have popTx that looks like

```go
import "github.com/BOPR/config"

func (db *DB) PopTxs() (txs []Tx, err error) {
// do something with config.GlobalCfg.TxsPerBatch
}
```
We should make it
```go
func (db *DB) PopTxs(config: GlobalCfg) (txs []Tx, err error) {
// do something with config.TxsPerBatch
}
```
or even better
```go
type DB struct {
	Instance *gorm.DB
	Logger   log.Logger
        config    config.GlobalCfg
}
func (db *DB) PopTxs() (txs []Tx, err error) {
// do something with db.config.TxsPerBatch
}

```

**Comment by vaibhavchellani** [comment]
I really like the approaches mentioned [here](https://www.alexedwards.net/blog/organising-database-access)

**Comment by ChihChengLiang** [comment]
Fixed in #106
